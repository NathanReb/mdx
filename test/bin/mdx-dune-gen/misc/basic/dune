(rule
 (target dune_gen.ml)
 (action
  (with-stdout-to
   %{target}
   (run ocaml-mdx dune-gen --prelude %{dep:prelude.ml} --prelude
     %{dep:prelude2.ml} --directory /foo/bar --directory /foo/foo))))

(rule
 (alias runtest)
 (action
  (diff %{dep:dune.gen.expected} %{dep:dune_gen.ml})))
