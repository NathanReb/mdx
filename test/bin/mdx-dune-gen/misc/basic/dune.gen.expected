let file = Sys.argv.(1)
let prelude = ["prelude.ml";"prelude2.ml"]

let directives = Mdx_top.directories ["/foo/bar";"/foo/foo"]

let _ = Mdx_test.run_exn_defaults
  ~file
  ~prelude
  ~directives
